### PR TITLE
All the goodies, Headers, copyright, mouse usage.

### PR DESCRIPTION
**Pull request by Malex**
This time preserving you code style, hope you like it and wish to actually make the pull

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,8 @@
-MIT License
```
Comment (shobrook) [inline LICENSE]: Not sure why the license was altered...

File: statcode/statcode.py [hunk 2]
```diff
@@ -109,6 +100,10 @@ def render(self, size, focus=False):
 
         return canv
 
+    def mouse_event(self, size, event, button, col, row, focus):
```
Comment (shobrook) [inline statcode/statcode.py]: Awesome! I've been trying to figure this out on a variety of urwid projects now, thanks.

File: LICENSE (hunk 1 above)
Comment (Malex) [reply LICENSE]: By mistake I guess, reverting it right now

**Comment by Malex** [comment]
Should be good now

**Pull request merged**